config/jobs/kubernetes-sigs/release-notes/release-notes-presubmits.yaml

```diff
@@ -162,10 +160,10 @@ presubmits:
         resources:
           limits:
             cpu: 4
-            memory: 6Gi
+            memory: 7.5Gi
           requests:
             cpu: 4
-            memory: 6Gi
+            memory: 7.5Gi
```

Comment:
i'm certain we have more than 2GB/core on these hosts, I suggest bumping to 8 but won't block on that
/lgtm
/approve
/hold
(you can comment `/hold cancel` if you'd prefer to go ahead, or push an update with 8gb and we'll re-lgtm after that)

Reply:
ok, I pushed with 8gb